src/test/java/org/xbill/DNS/SetResponseTest.java

```diff
@@ -77,21 +70,13 @@ void test_ctor_1arg()
     @Test
     void test_ctor_1arg_toosmall()
     {
-	try {
-	    new SetResponse(-1);
-	    fail("IllegalArgumentException not thrown");
-	}
-	catch(IllegalArgumentException a){}
+    	assertThrows(IllegalArgumentException.class, () -> new SetResponse(-1));
```

Comment:
There's a lot more to cleanup like this (unnecessary boxing, StringBuffer -> StringBuilder (already done, just not pushed yet)), but please let us do it at least _distinct_ commits.